battleship.py

```diff
@@ -42,9 +50,11 @@ def check_availability(target, grid):
         return False
 
 
-def check_hit(target, grid):
-    pass
-    # TODO: implement check_hit
+def check_hit(target, ships):
+    if target in ships:
+        return True
+    else:
+        return False
 
 
 def fire(target, grid):
```

Comment:
We will need to add a ships parameter here now to pass the ships to the check_hit function.